lib/flagship/features.rb

```diff
@@ -0,0 +1,17 @@
+class Flagship::Features < Array
+  def tagged(tags)
+    Flagship::Features.new(select{|feature| tags.any?{|tag, val| feature.tags[tag] == val}})
```

Comment:
You don't think an OR is useful? By the way, there's an alternative to `tagged_all`...

```ruby
Flagship.features.tagged(tag_a: true).tagged(tag_b: true)
```